### PR TITLE
Clarify value validity

### PR DESCRIPTION
**Pull request by Veykril**
Closes https://github.com/ferrocene/specification/issues/167

### CODE REVIEW

File: src/glossary.rst [hunk 1]
```diff
@@ -5324,6 +5324,15 @@ recognize :t:`[escaped character]s`.
...
 See :s:`RawStringLiteral`.
 
+.. _fls_sAe1HaaVSPvP:
+
+reachable control flow path
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+:dp:`fls_IxrvzuBg8j3E`
+A :dt:`reachable control flow path` is a control flow path that can be
+taken by the execution of a program to a given position in the program.
```
Comment (Dajamante) [inline src/glossary.rst:5334]: "from a given position in the program" seems more logical to me.

**Review by Dajamante — APPROVED** [review]

File: src/glossary.rst (hunk 1 above)
Comment (Veykril) [reply src/glossary.rst]: Rephrased it

**Comment by Dajamante** [comment]
bors merge

**Review by Dajamante — APPROVED** [review]

**Comment by Dajamante** [comment]
bors merge

**Pull request merged**